### PR TITLE
943 fulfilment mappings ind response print products

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

I've added in new fulfilment mappings for the individual UAC response print letters.

# What has changed

- Added test for new fulfilment codes

# How to test?

- Run ATs with other branches

# Links

[Trello](https://trello.com/c/2pVJow0S)

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[Fulfilment mappings - Ind Response Print Products (5)](https://trello.com/c/2pVJow0S/943-fulfilment-mappings-ind-response-print-products-5)

File: acceptance_tests/features/fulfilment_request.feature [hunk 1]
```diff
@@ -105,6 +105,25 @@ Feature: Handle fulfilment request events
...
+    And the fulfilment request event is logged
+
+    Examples: Individual Response Questionnaires fulfilment codes
+      | fulfilment code |
+      | P_UAC_UACIP1    |
+
+    @regression
+    Examples: Individual Response Questionnaires fulfilment codes
+      | fulfilment code |
+      | P_UAC_UACIP1    |
```
Comment (LeoKHoward) [inline acceptance_tests/features/fulfilment_request.feature]: Duplicated test

File: acceptance_tests/features/fulfilment_request.feature [hunk 2]
```diff
@@ -105,6 +105,25 @@ Feature: Handle fulfilment request events
...
 
+  Scenario Outline: Generate print files and log events for individual UAC print fulfilment letter requests
+    Given sample file "sample_1_english_HH_unit.csv" is loaded
+    And messages are emitted to RH and Action Scheduler with [01] questionnaire types
+    When an individual print fulfilment request "<fulfilment code>" is received by RM
+    Then a new individual child case for the fulfilment is emitted to RH and Action Scheduler
+    And correctly formatted individual UAC print responses are created with "<fulfilment code>"
+    And the fulfilment request event is logged
+
+    Examples: Individual Response Questionnaires fulfilment codes
```
Comment (LeoKHoward) [inline acceptance_tests/features/fulfilment_request.feature]: May be worth removing Questionnaires from the example title so its not misleading...this is supplementary documentation and not a questionnaire

Comment (ryangrundy7) [reply acceptance_tests/features/fulfilment_request.feature]: Done!

File: acceptance_tests/features/fulfilment_request.feature (hunk 1 above)
Comment (ryangrundy7) [reply acceptance_tests/features/fulfilment_request.feature]: Thought I deleted it! All done 👍

**Review by LeoKHoward — APPROVED** [review]
All good and passes

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Pull request merged**